controller/src/main/java/org/jboss/as/controller/management/Capabilities.java

```diff
@@ -40,4 +40,6 @@
 
     public static final String SSL_CONTEXT_CAPABILITY = "org.wildfly.security.ssl-context";
 
+    public static final String MANAGEMENT_SECURITY_REALM_CAPABILITY = "org.wildfly.core.management.security.realm";
```

Comment:
All the usages in full go through SecurityRealm.ServiceUtil.createServiceName so we can easily rename